src/map/clif.c

```diff
@@ -3754,11 +3789,7 @@ void clif_tradeadditem(struct map_session_data* sd, struct map_session_data* tsd
 {
 	int fd;
 	unsigned char *buf;
-#if PACKETVER < 20100223
-	const int cmd = 0xe9;
-#else
-	const int cmd = 0x80f;
-#endif
+	const int cmd = tradeaddType;
```

Comment:
Since you're at it, you could just remove this `cmd` constant now, and use tradeaddType directly below?